tools/region-parser/region-parser.ts

```diff
@@ -0,0 +1,127 @@
+const blockC = require('./region-matchers/block-c');
+const html = require('./region-matchers/html');
+const inlineC = require('./region-matchers/inline-c');
+
+export type Region = {lines: string[], open: boolean};
+export type RegionMap = {[regionName: string]: Region};
+
+export function regionParser(contents: string, fileType: string) {
+    return regionParserImpl(contents, fileType);
+}
+
```

Comment:
It seems like the indentation is off on this file. It's using 4 spaces instead of 2.